qi-lib/flow/core/compiler/deforest/cps.rkt

```diff
@@ -251,14 +257,14 @@
 
   ;; Transformers
 
-  (define-inline (map-cstream-next f next)
+  (define-inline (map-cstream-next f next ctx src)
```

Comment:
If unused, probably we should mark them as _ctx _src (or use them). This occurs throughout.

Reply:
Will look into this after merging this into the integration branch.